### PR TITLE
Cleanup Andrew Poelstra talk on secure Schnorr signatures

### PR DESCRIPTION
**Pull request by michaelfolkson**

### CODE REVIEW

**Comment by michaelfolkson** [comment]
Andrew doesn't have a Twitter account (to add to the Twitter JSON file) and this was just a cleanup anyway

**Comment by kouloumos** [comment]
> Andrew doesn't have a Twitter account (to add to the Twitter JSON file) and this was just a cleanup anyway

The bot doesn't tweet for modified transcripts anyway (should it tweet?). I'll try to read and merge this today.

**Comment by michaelfolkson** [comment]
> The bot doesn't tweet for modified transcripts anyway (should it tweet?)

Nah. Otherwise a typo fix would trigger the bot.

File: mit-bitcoin-expo/mit-bitcoin-expo-2019/signature-scheme-security-properties.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
-title: Signature Scheme Security Properties
+title: Andrew Poelstra - Secure Signatures (2019-03-09)
 transcript_by: Bryan Bishop
 categories: ['conference']
-tags: ['security', 'schnorr']
+tags: ['schnorr', 'multisig']
+speakers: ['Andrew Poelstra']
+date: 2019-03-09
```
Comment (adamjonas) [inline mit-bitcoin-expo/mit-bitcoin-expo-2019/signature-scheme-security-properties.md:7]: this needs to be removed. see #188

Comment (michaelfolkson) [reply mit-bitcoin-expo/mit-bitcoin-expo-2019/signature-scheme-security-properties.md]: Pushed an additional commit removing the date from the header

**Comment by adamjonas** [comment]
Up do you if you want to add back in the date

**Comment by michaelfolkson** [comment]
Ok, removed the commit that deleted the date in the header.

File: mit-bitcoin-expo/mit-bitcoin-expo-2019/signature-scheme-security-properties.md (hunk 1 above)
Comment (michaelfolkson) [reply mit-bitcoin-expo/mit-bitcoin-expo-2019/signature-scheme-security-properties.md:7]: Now removed that commit

**Pull request merged**